tests/Controller/CRUDControllerTest.php

```diff
@@ -3300,6 +3253,31 @@ public function testBatchActionWithoutConfirmation(): void
         $this->assertSame('list', $result->getTargetUrl());
     }
 
+    public function testBatchActionMethodNotExist(): void
```

Comment:
This method should be kept in its original place.

Reply:
Fixed